test.js

```diff
@@ -0,0 +1,6 @@
+console.log('foo');
+console.log('foo')
+    console.log('foo')
+    function a(foo) {
+      foo()
+  }
```

Comment:
__[development][eslint]__
* Error - Expected indentation of 4 spaces but found 2. (indent)

<!-- Comment by [SideCI](https://sideci.com) -->